proposals/lip-0057.md

```diff
@@ -1469,6 +1469,20 @@ Here `x` is the BFT weight at which the elements `validatorList[0].bftWeight, ..
 
 More functions might be made available during implementation.
 
+#### getActiveValidatorsNumber
```

Comment:
The word order is a bit odd, I would call it `getNumberOfActiveValidators`.

Reply:
done.